Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -46,6 +46,7 @@ def cmd_output(*args)
       ENV["HOMEBREW_BREW_FILE"] = HOMEBREW_PREFIX/"bin/brew"
       ENV["HOMEBREW_INTEGRATION_TEST"] = cmd_id_from_args(args)
       ENV["HOMEBREW_TEST_TMPDIR"] = TEST_TMPDIR
+      ENV["HOMEBREW_NO_GITHUB_API"] = "1"
```

Comment:
I think no tests shouldn't require you to be online to pass but it's currently only integration tests that have this issue. I could instead add this to `brew tests` and apply it globally but I'd rather the flag be `--online` rather than `--offline` for that reason.